connector/src/main/scala/com/microsoft/kusto/spark/common/KustoOptions.scala

```diff
@@ -35,7 +35,7 @@ trait KustoOptions {
   val KUSTO_TIMEOUT_LIMIT: String = newOption("timeoutLimit")
 }
 
-case class KustoCoordinates(cluster: String, database: String, table: Option[String] = None)
+case class KustoCoordinates(clusterUrl: String, clusterAlias:String, database: String, table: Option[String] = None)
```

Comment:
Why do we need both clusterAlias and clustyerUrl?

Reply:
We use alias for logs 
Before we were just keeping the alias but you cant construct the url from just the alias

Reply:
Alias is used also as key for the cached client